features/notifications/utils/onInteractWithNotification.ts

```diff
@@ -58,40 +55,13 @@ export const onInteractWithNotification = async (
     | undefined;
 
   if (conversationTopic) {
-    // todo(lustig): zod verification of external payloads such as those from
-    // notifications, deep links, etc
+    await waitForXmtpClientHydration();
```

Comment:
Turns out all the crap I was doing before in the previous diff uh was done in client hydration so it's not needed here